sale_partner_incoterm/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Default sales incoterm per partner
+==================================
+
+This module adds a field on the Sales & Purchases tab of the partner form where
+you can register the default incoterms for new sales orders for this customer.
+
+The field is only visible for customers. It is considered a commercial field,
+so it is synchronized between the company partner and all its contacts.
+
+When the partner is selected on a new quotation, the incoterm is retrieved from
+the partner record.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{sale-workflow}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{sale-workflow}/issues/new?body=module:%20
+{sale_partner_incoterm}%0Aversion:%20
+{8.0.1.0.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
This is only to have a clue about Odoo version, not module version, because the most of the times if we put the module version, we'll end with an incorrect one because contributors forget to change this data (it happens also in manifest, you know...)